webapp/google_calendar.py

```diff
@@ -0,0 +1,60 @@
+from google.oauth2 import service_account
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+import os
```

Comment:
Ideally should be at the top. https://peps.python.org/pep-0008/#imports.

You can use a plugin like isort, which is available in the vscode extensions to automatically do this for you.